app_dart/lib/src/request_handlers/push_engine_status_to_github.dart

```diff
@@ -99,4 +102,30 @@ class PushEngineStatusToGithub extends ApiRequestHandler<Body> {
     }
     return GithubBuildStatusUpdate.statusSuccess;
   }
+
+  Future<void> _insertBigquery(String buildStatus) async {
+    // Define const variables for [BigQuery] operations.
+    const String projectId = 'flutter-dashboard';
+    const String dataset = 'cocoon';
+    const String table = 'EngineBuildStatus';
+
+    final TabledataResourceApi tabledataResourceApi = await config.createTabledataResourceApi();
+    final List<Map<String, Object>> requestRows = <Map<String, Object>>[];
+
+    requestRows.add(<String, Object>{
+      'json': <String, Object>{
+        'Timestamp': DateTime.now().millisecondsSinceEpoch,
```

Comment:
Does this need to be UTC?

Reply:
We have been using now() for all cocoon time related values. I know GoB commit time, luci bigquery timestamp are using UTC. 
But this should not be an issue. We can easily adjust the time when we present the results in dashboard.